modules/quadbin/bigquery/README.md

```diff
@@ -1,3 +1,3 @@
 # Quadbin module for BigQuery
 
-You can learn more about quadbins in the [Overview section](/analytics-toolbox-bigquery/overview/spatial-indexes/#quadbin) of the documentation.
\ No newline at end of file
+You can learn more about quadbins in the [Overview section](/analytics-toolbox-bigquery/overview/spatial-indexes/#quadbin) of the documentation.
```

Comment:
I guess this is a branch of `feature/sc-238749/add-quadbin-module-all-clouds-bigquery` and latest changes were not taken so it consider the files in bigquery folder as added code.